package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/olegskl/gulp-stylelint",
   "engines": {
-    "node": ">=15.14.0"
+    "node": ">=12.0.0"
```

Comment:
Could you use the same versions as stylelint has?
https://github.com/stylelint/stylelint/blob/main/package.json#L194

Reply:
@onigoetz ^^

Reply:
I added the version to the suggestion.